ecs-init/config/os_family_linux.go

```diff
@@ -0,0 +1,88 @@
+//go:build linux
+// +build linux
+
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package config
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/cihub/seelog"
+)
+
+const (
+	unsupportedOSFamily = "linux"
+	OSFamilyEnvVar      = "ECS_OS_FAMILY"
+	osReleaseFilePath   = "/etc/os-release"
+)
+
+var execCommand = exec.Command
+
+func GetLinuxOSFamily() string {
+	cmd := execCommand("cat", osReleaseFilePath)
```

Comment:
instead of execing an external command, it would be safer just use os.ReadFile here